### ISSUE TITLE
init: Cookie config should take the same options as feathers-authentication.

### ISSUE THREAD

**Issue by suitupalex** [feathersjs/authentication-client#53]
TLDR: Cookie configuration should take in the same options as the service configuration. Also some module should clear the cookie so that we can more easily configure things like `domain` and `secure`.

I'll gladly make a PR if the proposed solution is sound!

### Steps to reproduce
Configure a feathers service to use a custom cookie configuration on a separate subdomain, e.g.:

```js
// For example, an API service running on https://api.staging.example.com.
app.configure({
  cookie: {
    domain: '.staging.example.com'
  , enabled: true
  , secure: true
  }
, ...
})
```

Setup a feathers client with feathers-authentication on a different domain (e.g. https://staging.example.com). Login, then try to logout.

### Expected behavior
`feathers-jwt` cookie should be cleared.

### Actual behavior
`feathers-jwt` cookie is not cleared.

### System configuration
For my specific use case, my feathers client is hosted on Heroku and is located at https://api.staging.athletes.gg, However, my website is hosted through Netlify and is located at https://staging.athletes.gg. The cookie works if I use `.staging.athletes.gg` or `.athletes.gg`, but since `utils.clearCookie` [does not take in domain or secure parameters like the service does](https://github.com/feathersjs/feathers-authentication-client/blob/master/src/utils.js#L40), the cookie does not get cleared.

My current workaround is to use `js-cookie` to clear the cookie when `app.logout` resolves:

```
app.logout().then(() => {
  Cookies.remove('feathers-jwt', {
    domain: '.staging.athletes.gg'
  , secure: true
  })
})
```

I'd recommend using a similar module to handle cookie deletion and update the configuration parsing.

**Module versions** (especially the part that's not working):
Server Side:
**├─ feathers-authentication-client@0.3.2
├─ feathers-authentication-jwt@0.3.1
├─ feathers-authentication-oauth1@0.2.4
├─ feathers-authentication-oauth2@0.2.4
├─ feathers-authentication@1.2.3**
├─ feathers-commons@0.8.7
├─ feathers-errors@2.5.0
├─ feathers-hooks@2.0.1
├─ feathers-memory@1.1.0
├─ feathers-mongodb@2.9.0
├─ feathers-query-filters@2.1.2
├─ feathers-rest@1.7.2
├─ feathers-socket-commons@2.4.0
├─ feathers-socketio@2.0.0
├─ feathers@2.1.2

Client Side:
**├─ feathers-authentication-client@0.3.2**
├─ feathers-client@2.2.0
├─ feathers-commons@0.8.7
├─ feathers-errors@2.7.0
├─ feathers-hooks@2.0.0
├─ feathers-primus@2.1.0
├─ feathers-rest@1.7.2
├─ feathers-socket-commons@2.4.0
├─ feathers-socketio@1.5.2
├─ feathers@2.1.1

**NodeJS version**:
v6.10.1

**Operating System**:
Windows Linux Subsystem (Ubuntu)

**Browser Version**:
Chrome 58.0.3029.110

**React Native Version**:
N/A

**Module Loader**:
Webpack 2

**Comment by cdimitroulas** [comment]
app.logout() is supposed to clear the cookie but it looks like it is not working. See [this](https://github.com/feathersjs/authentication-client/blob/master/lib/passport.js#L222) line.

**Comment by daffl** [comment]
Cookies are no longer used in Feathers v4 authentication. Please see the [migration guide](https://crow.docs.feathersjs.com/migrating.html) for more information. Closing this issue in order to archive this repository. Related issues can be opened at the new code location in the [Feathers main repository](https://github.com/feathersjs/feathers).
